glam/api/views.py

```diff
@@ -80,6 +82,8 @@ def get_firefox_aggregations(request, **kwargs):
     product = "firefox"
     channel = kwargs.get("channel")
     model_key = f"{product}-{channel}"
+    product_details = requests.get(PRODUCT_DETAILS_URLS[product]).json()
```

Comment:
It seems that now we call this API on each GET request. Would it visibly slow down the application? I think caching these values might make sense (would need to be periodically refreshed - either here or asynchronously). What do you think?